plugins/modules/clustering/nomad/nomad_job_info.py

```diff
@@ -0,0 +1,152 @@
+#!/usr/bin/python
+# coding: utf-8 -*-
+
+# (c) 2020, FERREIRA Christophe <christophe.ferreira@cnaf.fr>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = '''
+---
+module: nomad_job_info
+author: FERREIRA Christophe (@chris93111)
+version_added: "1.2.0"
+short_description: Get Nomad Jobs info
+description:
+    - Get info for one Nomad job.
+    - List Nomad jobs.
+requirements:
+  - python-nomad
+options:
+    host:
+      description:
+        - FQDN of Nomad server.
+      required: true
+      type: str
+    use_ssl:
+      description:
+        - Use TLS/SSL connection.
+      type: bool
+      default: true
+    timeout:
+      description:
+        - Timeout (in seconds) for the request to Nomad.
+      type: int
+      default: 5
+    validate_certs:
+      description:
+        - Enable TLS/SSL certificate validation.
+      type: bool
+      default: true
+    client_cert:
+      description:
+        - Path of certificate for TLS/SSL.
+      type: path
+    client_key:
+      description:
+        - Path of certificate's private key for TLS/SSL.
+      type: path
+    namespace:
+      description:
+        - Namespace for Nomad.
+      type: str
+    token:
+      description:
+        - ACL token for authentification.
+      type: str
+    name:
+      description:
+        - Name of job for Get info.
+      type: str
+notes:
+  - C(check_mode) is supported
+seealso:
+  - name: Nomad jobs documentation
+    description: Complete documentation for Nomad API jobs.
+    link: https://www.nomadproject.io/api-docs/jobs/
+'''
+
+EXAMPLES = '''
+- name: Get info for job awx
+  community.general.nomad_job:
+    host: localhost
+    name: awx
```

Comment:
```suggestion
    name: awx
  register: result
```
I think for _info plugins it is good to also add the `register` keyword, since that's how they are usually used.